packages/editor-ui/src/views/SignupView.vue

```diff
@@ -75,21 +79,25 @@ export default mixins(
 		};
 	},
 	async mounted() {
-		const inviterId = this.$route.query.inviterId;
-		const inviteeId = this.$route.query.inviteeId;
+		const inviterId = this.$route.query.inviterId as string || '';
```

Comment:
need to be careful here

Reply:
This one is not as trivial as it seems. Route params can be `string` or `string[]` (in case same param is specified multiple times).
So  I guess our options are either to force `string` type as is or to do it but also checking if param type is not `Array`. I updated this code to have array check.